### PR TITLE
fix long description name may overflow the Image item

### PR DESCRIPTION
**Pull request by BangMach**
By using Balancer from react-wrap-balancer library to fix long item's name
Also: 
Install Playwright on home computer and run a few UI test, discover long item description will overflow on the Image UI

### CODE REVIEW

**Comment by BangMach** [comment]
previously, trying to get playwright to work on personal Macbook computer, and delete some  unnecessary test case

File: src/data/food_options.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [
   {
     "id": "hot",
-    "name": "Hot",
-    "description": "Hot coffee",
+    "name": "Hot Mocha Coffee",
```
Comment (vietanhdev) [inline src/data/food_options.json:4]: @BangMach Products are now stored in the database. This file (`food_options.json`) will be removed.

File: tests/test-13.spec.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { expect, test } from '@playwright/test';
+
+test('test', async ({ page }) => {
+  await page.goto('https://cafe.vietanh.dev/en');
```
Comment (vietanhdev) [inline tests/test-13.spec.ts:4]: Don't hardcode the URL. This is for the test, so run a local version.

File: src/app/[lng]/pick-item-options/[itemId]/page.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+import Image from "next/image";
+import XButton from "@/components/buttons/XButton";
 import { useRouter } from "next/navigation";
-const itemsOptions = require("@/data/food_options.json");
-import { useTranslation } from "@/app/i18n/client";
 import { useSession } from "next-auth/react";
-import XButton from "@/components/buttons/XButton";
+import { useTranslation } from "@/app/i18n/client";
 
+const itemsOptions = require("@/data/food_options.json");
```
Comment (vietanhdev) [inline src/app/[lng]/pick-item-options/[itemId]/page.js:12]: All food options are pulled from database now. Please pull the latest code from master branch.

**Review by vietanhdev — CHANGES_REQUESTED** [review]

**Comment by vietanhdev** [comment]
@BangMach Try to pull the code from `master` branch and resolve all conflicts.
Be sure not to use `food_options.json`. You can delete that file.

File: src/data/food_options.json (hunk 1 above)
Comment (BangMach) [reply src/data/food_options.json:4]: okayy I will update this changes in my next pull request

File: src/app/[lng]/pick-item-options/[itemId]/page.js (hunk 3 above)
Comment (BangMach) [reply src/app/[lng]/pick-item-options/[itemId]/page.js:12]: thank you. I will just create a new branch from master and update from it then, seem like this branch is a little outdated

**Pull request closed without merging**